bears/documentation/DocGrammarBear.py

```diff
@@ -0,0 +1,117 @@
+import shutil
+
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocBaseClass import (
+    DocBaseClass)
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+
+class DocGrammarBear(DocBaseClass, LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    REQUIREMENTS = {PipRequirement('language-check', '1.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/132564'
+    CAN_DETECT = {'Documentation', 'Spelling', 'Grammar'}
```

Comment:
Yeah, because if we can fix something, we have already detected that thing. :smile: